GalaxyZooTouchTable/GalaxyZooTouchTable/Models/ClassificationSummary.cs

```diff
@@ -17,6 +18,8 @@ public ClassificationSummary(string subjectLocation, ClassificationCounts counts
             SubjectLocation = subjectLocation;
             SummaryString = summaryString;
             TotalVotes = counts.Total;
+            if (TotalVotes > 10)
+                VoteLimit = counts.Total;
```

Comment:
Useful. I like that the VoteLimit takes into account the possibility that there are more votes than that limit.